docs/setup.rst

```diff
@@ -0,0 +1,95 @@
+Setting Up Corobo
+=================
+
+The following documentation illustrates the process to
+install corobo and interface it with the Gitter 
+(or any other backend) API.
+
+Installation
+------------
+- To install corobo, first you need to install 
+  python (version 3.6 or more) and set up a virtual environment. 
+  This can be done as follows:
+  ::
+    python3 -m venv env
+    cd env
+    source bin/activate ---->  For Linux
+    \Scripts\activate   ---->  For Windows
+
+- Next, you can clone the corobo package via git and 
+  install the requirements (dependent packages).
+  ::
+    git clone https://github.com/coala/corobo.git
+    cd corobo
+    pip install -r requirements.txt
+
+Configuring the backend
+-----------------------
+
+- Clone the err-backend-gitter repository in your "corobo" folder.
+  ::
+    pip3 install git+https://github.com/errbotio/\
+    err-backend-gitter@14a428bdd0597b473605264c822855da979bd916  
```

Comment:
This is not the way we install gitter backend. It is a part of the git submodules so the user just needs to `update` the git submodule.